test/run-unittest.d

```diff
@@ -107,5 +73,16 @@ int main(string[] args)
 		}
 	}
 
+	foreach(DirEntry script; dirEntries(curr_dir, (args.length > 1) ? args[1] : "*.script.d", SpanMode.shallow))
+	{
+		const min_frontend = script.name ~ ".min_frontend";
+		if (exists(min_frontend) && frontend.length && frontend < min_frontend.readText) continue;
```

Comment:
```suggestion
		if (frontend.length && exists(min_frontend) && frontend < min_frontend.readText) continue;
```

Since `frontend.length` is constant through the program, and does not require file IO, put this condition first.